src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -58,16 +59,28 @@ public StudyController(StudyService studyService, NetworkStoreService networkSto
         this.networkModificationService = networkModificationService;
     }
 
-    static class MyEnumConverter extends PropertyEditorSupport {
+    static class MyEnumConverter<E extends Enum<E>> extends PropertyEditorSupport {
```

Comment:
Too complicated ?
I think is better to do the conversion in the service in order to facilitate reading

Reply:
I do not agree. This is REST interface thing, not service.